packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -372,6 +374,10 @@ export const getInterpolation = (expression: t.Expression, meta: Metadata): t.Ex
   if (t.isIdentifier(expression)) {
     const binding = meta.parentPath.scope.getBinding(expression.name);
     const resolvedBinding = resolveBindingNode(binding, meta);
+    if (binding?.path.node === expression) {
```

Comment:
hehe 😛. Breaking the circular cycle with reference check. Nice.

Reply:
yeah, elegant!